### ISSUE TITLE
Gecko IPC subsystem prints too many errors

### ISSUE THREAD

**Issue by victorsowa** [mozilla/geckodriver#1605]
## System

* Version: 0.24.0
* Platform: Ubuntu 18.04
* Firefox: 68.0.2
* Selenium:  3.141.0 with Python3

## Trace-level log

```1566297958715	mozrunner::runner	INFO	Running command: "/usr/bin/firefox" "-marionette" "-headless" "-foreground" "-no-remote" "-profile" "/tmp/rust_mozprofile.5oteSpr6d6eA"
*** You are running in headless mode.
1566297960108	addons.webextension.screenshots@mozilla.org	WARN	Loading extension 'screenshots@mozilla.org': Reading manifest: Invalid extension permission: mozillaAddons
1566297960108	addons.webextension.screenshots@mozilla.org	WARN	Loading extension 'screenshots@mozilla.org': Reading manifest: Invalid extension permission: telemetry
1566297960109	addons.webextension.screenshots@mozilla.org	WARN	Loading extension 'screenshots@mozilla.org': Reading manifest: Invalid extension permission: resource://pdf.js/
1566297960109	addons.webextension.screenshots@mozilla.org	WARN	Loading extension 'screenshots@mozilla.org': Reading manifest: Invalid extension permission: about:reader*
1566297966399	Marionette	INFO	Listening on port 33577
1566297966481	Marionette	WARN	TLS certificate errors will be ignored for this session
console.error: BroadcastService: 
  receivedBroadcastMessage: handler for
  remote-settings/monitor_changes
  threw error:
  Message: Error: Polling for changes failed: http://%(server)s/dummy/blocklist//buckets/monitor/collections/changes/records?_expected=%221566233826256%22 is not a valid URL..
  Stack:
    remoteSettingsFunction/remoteSettings.pollChanges@resource://services-settings/remote-settings.js:201:13

1566298097983	Marionette	INFO	Stopped listening on port 33577
[Parent 47, Gecko_IOThread] WARNING: pipe error: Broken pipe: file /builds/worker/workspace/build/src/ipc/chromium/src/chrome/common/ipc_channel_posix.cc, line 727

###!!! [Parent][MessageChannel] Error: (msgtype=0xA10001,name=PVsync::Msg_Notify) Channel error: cannot send/recv
```

The row:
`###!!! [Parent][MessageChannel] Error: (msgtype=0xA10001,name=PVsync::Msg_Notify) Channel error: cannot send/recv` 
is the repeated millions of times.

**Comment by andreastt** [comment]
This log message is from the IPC subsystem in Firefox, hence not a geckodriver bug. The log file’s size presumably depends on how long the process runs for.

@whimboo Do you know if there’s a way we can disable these logs? If they are printed so often, it can quickly escalate to a usage issue of geckodriver.

**Comment by whimboo** [comment]
Sorry but I don't know how to stop IPC logging. But maybe @jld could know if there is a preference or whatever.

**Comment by andreastt** [comment]
Related: https://bugzilla.mozilla.org/show_bug.cgi?id=1399441#c11

tl;dr: we should turn off Firefox stdout altogether and stream relevant logs back to the client (geckodriver) for printing.

**Comment by TurdOnTheRun** [comment]
Having the same problem.

**Comment by whimboo** [comment]
Dupe of issue #1046.
